### PR TITLE
Merging CAN to main

### PR DESCRIPTION
**Pull request by Pandabear1125**

### CODE REVIEW

**Comment by StudentKyle** [comment]
Closing and opening to run doxygen test

**Comment by StudentKyle** [comment]
re-running test

**Comment by StudentKyle** [comment]
It looks like there are so many errors that the comment bot fails.

**Comment by StudentKyle** [comment]
Documentation looks good

**Review by StudentKyle — CHANGES_REQUESTED** [review]
Documentation looks good, but file structure is not correct with how the rest of the project is set up.

File: src/comms/rm_CAN.hpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+  /// @brief Sets output array to 0 and writes 0s to the CANs
+  /// @note Does issue a Write command to the CANs
+  void zero();
+
+  /// @brief Sets the output array message for the corresponding motor
+  /// @note Does not issue a Write command to the CANs
+  /// @param canID ID of the CAN which the motor is on, expects indexable ID value
+  /// @param motorID ID of the individual motor, expects indexable ID value
+  /// @param value A value in the range of []
```
Comment (StudentKyle) [inline src/comms/rm_CAN.hpp:58]: Incomplete docs

**Review by Rishi-H — APPROVED** [review]

**Review by StudentKyle — APPROVED** [review]
Looks good

**Pull request merged**